CONTRIBUTING.md

```diff
@@ -118,7 +118,7 @@ A useful shell alias would use the directory name as a scope:
 
 ```bash
 # add to your ~/.zshrc or ~/.bashrc
-alias lr='lerna run --stream --scope $(basename $PWD)'
+alias lr='lerna run --stream --scope $(node -p "require(\"./package.json\").name")'
```

Comment:
Suggestion: use `'` in the node script, so you don't have to escape the `"`s.

Reply:
Ah yeah.

Reply:
Oh no, that won't work because of the single quotes it's already in.